### PR TITLE
PythonAnywhere deploy: some tweaks and improvements based on experiences from first workshop

### PR DESCRIPTION
**Pull request by hjwp**
hopefully addresses #360 and a few other things...

### CODE REVIEW

File: en/deploy/README.md [hunk 1]
```diff
@@ -184,6 +184,8 @@ Just like you did on your own computer, you can create a virtualenv on PythonAny
     Successfully installed django-1.8 whitenoise-1.0.6
 
 
+> **Note** The pip install step can take a couple of minutes.  Patience, patience!  But if it takes more than 5 minutes, something is wrong.  Ask your coach.
```
Comment (bmispelon) [inline en/deploy/README.md]: nitpick: can we put "pip install" between backticks?

File: en/deploy/README.md [hunk 2]
```diff
@@ -291,13 +293,19 @@ We're all done!  Hit the big green **Reload** button and you'll be able to go vi
 
 ## Debugging tips
 
-If you see an error when you try to visit your site, the first place to look for some debugging info is in your **error log** -- you'll find a link to this on the PythonAnywhere web tab. See if there are any error messages in there. The most recent ones are at the bottom. Common problems include
+If you see an error when you try to visit your site, the first place to look for some debugging info is in your **error log** -- you'll find a link to this on the PythonAnywhere Web tab. See if there are any error messages in there; the most recent ones are at the bottom. Common problems include:
```
Comment (bmispelon) [inline en/deploy/README.md]: Is there a way to turn "PythonAnywhere Web tab" into a clickable link that takes you to the right place (assuming the reader is logged in)?

**Comment by bmispelon** [comment]
A few nitpicky comments but looks good otherwise.

Thanks, Harry.
Tharry.
:pray: :goat:

**Comment by hjwp** [comment]
http://i284.photobucket.com/albums/ll36/Bigsteve87/Gifs/AsYouWishPrincessBride.gif

**Comment by aniav** [comment]
Thank you! :zap: :zap: :zap:

**Pull request merged**